ModularTegustation/tegu_items/backstreets/templates.dm

```diff
@@ -28,7 +28,12 @@
 	room_id = "tunnels_backstreets"
 	mappath = "_maps/RandomRooms/backstreetlayout/tunnels_backstreets.dmm"
 
-/datum/map_template/random_room/backstreet_template/lcorp_ruin
-	name = "L-Corp Ruin Template"
-	room_id = "lcorp_ruin"
+/datum/map_template/random_room/backstreet_template/abandoned_lcorp
+	name = "Abandoned L-Corp Template"
+	room_id = "abandoned_lcorp"
 	mappath = "_maps/RandomRooms/backstreetlayout/lcorp_backstreets.dmm"
+
+/datum/map_template/random_room/backstreet_template/la_mancha_land
```

Comment:
Would like to ask to change this one's weight from 10 to 5, that means that it shows up half as often.

It's spectacular and flashy, but a little too different to be on par with the others.